3.10/indexing-index-basics.md

```diff
@@ -212,94 +208,54 @@ sort step:
+  the collection for documents that have the indexed attributes set. As the index will
   exclude documents for which the indexed attributes are `null` or not set, it can be
   used for optional attributes.
 
-- **non-unique skiplist index**: all documents in the collection will be indexed. This type
-  of index is not sparse. Documents that do not contain the index attributes or that have 
-  a value of `null` in the index attribute(s) will still be indexed. Duplicate key values 
+- **non-unique persistent index**: all documents in the collection will be indexed. This type
+  of index is not sparse. Documents that do not contain the index attributes or that have
+  a value of `null` in the index attribute(s) will still be indexed. Duplicate key values
   can occur and do not lead to unique constraint violations.
- 
-- **non-unique, sparse skiplist index**: only those documents will be indexed that have all
+
+- **non-unique, sparse persistent index**: only those documents will be indexed that have all
   the indexed attributes set to a value other than `null`. It can be used for optional
   attributes.
 
-The operational amortized complexity for skiplist indexes is logarithmically correlated
-with the number of documents in the index.
-
-Skiplist indexes support [indexing array values](#indexing-array-values) if the index
-attribute name is extended with a `[*]``.
-
--->
-
-Persistent Index
-----------------
-
-The persistent index is a sorted index with persistence. The index entries are written to
-disk when documents are stored or updated. That means the index entries do not need to be
-rebuilt from the collection data when the server is restarted or the indexed collection
-is initially loaded. Thus using persistent indexes may reduce collection loading times.
-
-The persistent index type can be used for secondary indexes at the moment. That means the
-persistent index currently cannot be made the only index for a collection, because there
-will always be the in-memory primary index for the collection in addition, and potentially
-more indexes (such as the edges index for an edge collection).
-
-The index implementation is using the RocksDB engine, and it provides logarithmic complexity
-for insert, update, and remove operations. As the persistent index is not an in-memory
-index, it does not store pointers into the primary index as all the in-memory indexes do,
-but instead it stores a document's primary key. To retrieve a document via a persistent
-index via an index value lookup, there will therefore be an additional O(1) lookup into 
-the primary index to fetch the actual document.
-
-As the persistent index is sorted, it can be used for point lookups, range queries and sorting
-operations, but only if either all index attributes are provided in a query, or if a leftmost 
-prefix of the index attributes is specified.
-
-<!-- TODO
+If a persistent index is created on an attribute that is missing in all or many
```

Comment:
Covered in the sections before already. I suggest removing this to shorten the section!